app/models/AppResult.java

```diff
@@ -75,6 +77,9 @@
     public static final String JOB_DEF_URL = "jobDefUrl";
     public static final String FLOW_DEF_URL = "flowDefUrl";
     public static final String APP_HEURISTIC_RESULTS = "yarnAppHeuristicResults";
+    public static final String RESOURCE_USAGE = "resourceUsage";
```

Comment:
How about renaming this to 
1. resourceUsed and resourceWasted or 
2. resourceUsage and resourceWastage?